<launch>
  <arg name="my_id" value="1"/>
  <arg name="odom_topic" value="/vins_fusion/imu_propagate"/>

  <!-- Launch ROS Package Template Node -->
  <node pkg="drone_detect" type="drone_detect" name="test_drone_detect" output="screen">
    <rosparam command="load" file="$(find drone_detect)/config/camera.yaml" />
    <rosparam command="load" file="$(find drone_detect)/config/default.yaml"/>
    <param name="my_id"      value="$(arg my_id)" />
    <param name="debug_flag" value="false" />
    <remap from="~odometry"   to="$(arg odom_topic)"/>
    <remap from="~depth" to="/camera/depth/image_rect_raw"/>
    <!-- <remap from="~camera_pose" to="/drone_$(arg my_id)_pcl_render_node/camera_pose"/> -->

    <!-- <remap from="~drone0" to="/drone_0_$(arg odom_topic)"/>
    <remap from="~drone1" to="/drone_1_$(arg odom_topic)"/>
    <remap from="~drone2" to="/drone_2_$(arg odom_topic)"/> -->

    <!-- <remap from="~drone1" to="/test/dynamic0_odom"/> -->
    <!-- <remap from="~drone2" to="/test/dynamic1_odom"/> -->

  </node>

</launch>
